addons/project/models/project_task.py

```diff
@@ -1608,3 +1608,10 @@ def get_unusual_days(self, date_from, date_to=None):
             datetime.combine(fields.Date.from_string(date_from), time.min).replace(tzinfo=UTC),
             datetime.combine(fields.Date.from_string(date_to), time.max).replace(tzinfo=UTC)
         )
+
+    def redirect_to_project_task_form(self):
```

Comment:
Can you rename the action like: `action_redirect_to_project_task_form`?
I prefer all actions name will start with `action_` 😇 

```suggestion
    def action_redirect_to_project_task_form(self):
```